templates/jenkins.j2

```diff
@@ -0,0 +1,33 @@
+NAME={{ jenkins_name }}
+JAVA={{ jenkins_java_location }}
+JAVA_ARGS="{{ (jenkins_java_opts + jenkins_java_user_opts) | join(" ") }}"
```

Comment:
I think that `jenkins_java_opts` is more than enough here. If someone wants to append something to the defaults, they can supply a new list that includes the previous defaults.